paths/graphql/02-first-use.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Use GraphQL for the First Time
+permalink: /graphql/first-use
+next-page: /graphql/first-query
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  When we introduced APIs, the technology that allows programs to interact with existing information, we briefly mentioned GraphQL. In this section, we are going to use a tool that enables you to experiment with GraphQL quickly and easily.
+
+  ### Navigating to GraphQL
+   1. [Navigate to the GraphQL explorer](https://developer.github.com/v4/explorer/).
+   1. Once the page has loaded, click the "Sign in with GitHub" button. If you don't have a GitHub account, [here are some instructions for getting an account setup](https://help.github.com/articles/signing-up-for-a-new-github-account/).  
+
+      After logging in, The GraphiQL interface displays a sample query in the left pane that should look like this:
+
+          # Type queries into this side of the screen, and you will
+          # see intelligent typeaheads aware of the current GraphQL type schema,
+          # live syntax, and validation errors highlighted within the text.
+
+          # We'll get you started with a simple query showing your username!
+          query {
+            viewer {
+              login
+            }
+          }
+
+   1. Click the "Execute Query" button (it looks like a Play button) to run the query that is in the left pane and display the results in the right pane.
+
+      Clicking "Execute Query" should present something that looks like this:
+
+
+          {
+            "data": {
+              "viewer": {
+                "login": "githubteacher"
+              }
+            }
+          }
+
+
+      Your result should look similar to the results displayed above, however, `githubteacher` should be replaced with your username (or login as the query displays).
+
+  1. Make a minor change to find the URL of your avatar on GitHub. Add the following code to your query, so it will return a URL to your avatar image.
+
+      ```
+      avatarUrl
+      ```
+
+      To see how to add the `avatarUrl` **field** click on "Tell me why" below :point_down:.
+
+show-me-how: "images/gifs/graphql/first-query.gif"
+
+tell-me-why: |
+
+  ### Using `avatarUrl`
+
+  At the end of the lesson, we provided an opportunity for you to experiment using the `avatarUrl` field. If you weren't able to get the query to run, or you just want to see how you would add additional fields to your query, you can make your query look like the following:
+
+      {
```

Comment:
I agree with @hollenberry's statement here. I'm not sure why we are adding an extra layer to the JSON.